File: python/transcribe_file_stream.py
from soniox.transcribe_file import transcribe_file_stream
from soniox.speech_service import SpeechClient


def main():
    with SpeechClient() as client:
        result = transcribe_file_stream(
            "../test_data/test_audio_long.flac",
            client,
            model="en_v2",
        )
        print("".join(w.text for w in result.words))


if __name__ == "__main__":
    main()

File: speech_to_text_legacy/csharp/TranscribeFileStream/TranscribeFileStream.cs
using Soniox.Types;
using Soniox.Client;
using Soniox.Client.Proto;

using var client = new SpeechClient();

var completeResult = await client.TranscribeFileStream(
    "../../test_data/test_audio_long.flac",
    new TranscriptionConfig
    {
        Model = "en_v2",
    });

Result result = (completeResult as SingleResult)!.Result;

var text = string.Join("", result.Words.Select(word => word.Text).ToArray());
Console.WriteLine("Text: " + text);
Console.WriteLine("Tokens:");
foreach (var word in result.Words)
{
    Console.WriteLine($"    '{word.Text}' {word.StartMs} {word.DurationMs}");
}